docs/api/browser-window.md

```diff
@@ -24,10 +24,11 @@ you can use the [Frameless Window](frameless-window.md) API.
 
 ## Showing window gracefully
```

Comment:
This is tangential from the focus of this PR, but this and the other changed heading lines feel like they need definite articles:
```suggestion
## Showing the window gracefully
```